Contentless/Program.cs

```diff
@@ -142,6 +202,12 @@ public static class Program {
         Console.Write("Done");
     }
 
+    private static string CalculateFullPathToLibrary(string libraryName, string referencesVersion)
+    {
+        var settings = Settings.LoadDefaultSettings(null);
+        return Path.Combine(SettingsUtility.GetGlobalPackagesFolder(settings), libraryName.ToLower(), referencesVersion, "tools", libraryName + ".dll");
```

Comment:
This seems a little arbitrary, are you 100% sure this is *always* the library path? Is there no way for users to change this in their system or NuGet config somehow?

Reply:
Now it reads user profile Nuget.Config, I have checked this (but it's single config file on my machine). Tomorrow I will face it more deeply. Also it read environment variables.
I see problem it does not read custom nuget config in project folder. I will fix it. I will search in project dir `NuGet.Config` and  let user override it via settings.
I have question about `tools` subfolder. I see you and `MonoGame.Extended` use it, but I'm not sure it's always correct. What do you think?

Reply:
That's the issue: I don't know either haha! I know that `tools` is the folder you're "meant" to use, but I don't know if that's because it's the only possible one, or because it's just a convention that people agreed on :(